tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/DebugInstructionTest.java

```diff
@@ -108,9 +108,6 @@ public void objectDebugInstructionTest() throws BallerinaTestException {
         // debugHitInfo = debugTestRunner.waitForDebugHit(10000);
         // Assert.assertEquals(debugHitInfo.getLeft(), new BallerinaTestDebugPoint(debugTestRunner.testEntryFilePath,
         // 33));
-        debugTestRunner.resumeProgram(debugHitInfo.getRight(), DebugTestRunner.DebugResumeKind.STEP_OVER);
```

Comment:
Why do we need to remove this?Can you check with @NipunaRanasinghe

Reply:
According to the requirement the last line (right brace) of the class definition also needed to be removed from the line number table. Previously that line was hit when we do step over or step in and removed with the new change. So the line#27 in the test not going to be hit by debug pointer.

Reply:
@warunalakshitha @gabilang yes thats correct. Need to remove L111-L113 as the list line wont be hit, and need to uncomment L103-L110 that was disabled due to this issue